lib/Standalone/ConvertTppToXsmm.cpp

```diff
@@ -115,13 +115,13 @@ struct ConvertTppBrgemmOp : public OpRewritePattern<BrgemmOp> {
     if (failed(ldaDim))
       return failure();
     int64_t lda = *ldaDim;
-    if (memrefA.getElementType().isBF16() && memrefA.getShape().size() == 3) {
-      auto divLdaDim = getLeadingDim(memrefA, 1);
+    if (memrefA.getElementType().isBF16() && memrefA.getShape().size() == 4) {
```

Comment:
It's unclear to me what's going on here. Can you add a comment to explain what are those dimensions and what `b` is in the end?

Reply:
Duplicate of @chelini comment, ignore it.